assets/prometheus-k8s/rules.yaml

```diff
@@ -873,6 +873,23 @@ spec:
           )
         )
       record: cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum
+    - expr: sum(node_role_os_version_machine:cpu_capacity_cores:sum{label_node_role_kubernetes_io_master="",label_node_role_kubernetes_io_infra=""}
+        or absent(node_role_os_version_machine:cpu_capacity_cores:sum{label_node_role_kubernetes_io_master="",label_node_role_kubernetes_io_infra=""})*0)
+        + (sum(node_role_os_version_machine:cpu_capacity_cores:sum{label_node_role_kubernetes_io_master="true"})
+        * ((max(cluster_master_schedulable == 1)*0+1) or (absent(cluster_master_schedulable
+        == 1)*0)))
+      record: workload:capacity_physical_cpu_cores:sum
```

Comment:
that's a beefy rule, leaving another pair of eyes from @openshift/openshift-team-monitoring

Reply:
It would be good to have unit test for this. Similar to what we have in https://github.com/openshift/cluster-monitoring-operator/blob/master/test/rules/bz1879520.yaml

Reply:
Ack